quay-enterprise/clair.md

```diff
@@ -0,0 +1,274 @@
+# Clair Setup
+
+<img src="img/Clair_horizontal_color.png" class="img-center" alt="Clair Security Scanner"/>
+
+<a href="https://github.com/coreos/clair">Clair</a> is an open source project for the static analysis
+of vulnerabilities in containers images. It can be used as the engine powering [Quay Security Scanner](security-scanning.md)
```

Comment:
s/containers/container/
"The Clair open source code is the engine that powers [Quay Security Scanner]()."